lightningd/closing_control.c

```diff
@@ -120,6 +124,12 @@ static void peer_closing_complete(struct channel *channel, const u8 *msg)
 	if (channel->state == CLOSINGD_COMPLETE)
 		return;
 
+	bitcoin_txid(channel->last_tx, &closing_txid);
+	billboard_msg = tal_fmt(channel,
```

Comment:
not sure if `channel` is the right level to allocate onto?

Reply:
No, that would be a (technical) memory leak. (Technical because this can only happen once per channel, so it's ultra constrained).  Unless you use "take()" below to hand it to channel_set_billboard, in whihc case the pattern is to allocate off NULL to make it clear it's going to be taken().

Reply:
cool, updated!